source/settings/plugin/acl.rst

```diff
@@ -42,6 +42,9 @@ ACLs can be enabled in ``dovecot.conf`` with:
     #acl_globals_only = yes
   }
 
+Global and per-user ACLs have different use cases, and both can be enabled at
```

Comment:
this paragraph seems bit random here.

Reply:
I can relate to that remark, but when I first read through the text, I thought that those two are in an either/or relation. I don't insist on keeping that sentence there, but perhaps you can suggest a better wording?

Reply:
Maybe just remove this paragraph?